code/modules/gear_presets/clf.dm

```diff
@@ -751,7 +751,7 @@
 
 	new_human.equip_to_slot_or_del(new /obj/item/storage/backpack/marine/engineerpack/ert, WEAR_BACK)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/gloves/combat, WEAR_HANDS)
-	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/combat, WEAR_FEET)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/marine/upp, WEAR_FEET)
```

Comment:
Redundant, CLF Synthetics already have knives in their face slot, see line 766

Reply:
I didn't realize because synths kept getting knocked down and dropping them